data-reconciliation-app/docs/adr/03-reconciliation-logic.md

```diff
@@ -57,40 +57,38 @@ Each record will have all members' opinions for it. So for each record the recon
 
 ### Voting Threshold
 
-Reconciliation logic is considered only if a specified number of opinions for a record is submitted. The `voting_threshold` determines if the record has received enough opinions to be reconciled. This will be configurable and can be set by members of the network, depending on the size of the network.
+Reconciliation logic is considered only if a specified number of users have submitted that record. The `voting_threshold` determines if the record has been submitted by enough users. This is configurable and can be set by members of the network, depending on the size of the network.
```

Comment:
Maybe add a link to where voting threshold is defined in the code?

Reply:
Added!